Join Us Card Component

mmart570 opened the pull request:
created Join Us folder with a page.tsx file to create the card (let me know if i have to fix this) 
<img width="1417" alt="Screenshot 2025-04-16 at 4 18 58 PM" src="https://github.com/user-attachments/assets/9f916260-ec75-41c0-bc70-f7275a5d3ddb" />

arieen17 commented on src/app/joinus/page.tsx:
```diff
@@ -0,0 +1,24 @@
+            Gain access to a strong network of healthcare professionals and
+            mentors who can guide you on your academic and career path.
+          </p>
+          <img
```
use the Image from next/Image instead of img

arieen17 commented on src/app/joinus/page.tsx:
```diff
@@ -0,0 +1,24 @@
+const MentorshipCard = () => {
```
you should create a component of a card to which you are pass it different variables of: an image and text
- then you would call it (3 times i believe for each different card) in the page for join us, instead of calling it directly

arieen17 reviewed (commented):
hhi good start, make sure you create a component then call then making it directly, and also unstage your example component!

tingtingtingtingtingtingtingtingtingtin commented on src/app/page.tsx:
```diff
@@ -1,3 +1,4 @@
+//import Example from "@/components/Example";
```
```suggestion

```

tingtingtingtingtingtingtingtingtingtin commented on src/components/joinus/mentorshipcard.tsx:
```diff
@@ -0,0 +1,22 @@
+const MentorshipCard = () => {
```
- call this component JoinUsCard
- it should take in the props arielle mentioned

tingtingtingtingtingtingtingtingtingtin commented on public/mentorshipcardicon.svg:
import a similar icon from react icons instead of directly getting the image

tingtingtingtingtingtingtingtingtingtin commented:
also use Next/Image instead of img

mmart570 commented:
<img width="2055" alt="Screenshot 2025-04-23 at 3 10 25 PM" src="https://github.com/user-attachments/assets/98fd429b-6cf8-4a8d-aa5d-ea8283dc3d0a" />
addressed comments, changed svg with temp react icon

mmart570 commented:
<img width="2056" alt="Screenshot 2025-04-23 at 4 44 47 PM" src="https://github.com/user-attachments/assets/fc4aa5d9-7dae-4853-b8d9-fb073d2a5d6f" />
finished rest of cards

tingtingtingtingtingtingtingtingtingtin commented on src/components/joinus/joincards.tsx:
```diff
@@ -0,0 +1,26 @@
+  return (
+    <div className="flex flex-wrap justify-center gap-8">
+      <div className="bg-naama-ivory-100 shadow-3xl flex w-3/4 flex-col items-center rounded-3xl p-5 text-wrap">
+        <div className="text-naama-blue-200 font-sans-nunito text-xl font-bold">
```
```suggestion
        <div className="text-naama-blue-200 font-nunito text-xl font-bold">
```

tingtingtingtingtingtingtingtingtingtin commented on src/components/joinus/joincards.tsx:
```diff
@@ -0,0 +1,26 @@
+        <div className="text-naama-blue-200 font-sans-nunito text-xl font-bold">
+          {title}
+        </div>
+        <p className="text-naama-blue-200 font-sans-nunito mt-3 text-center font-[400]">
```
```suggestion
        <p className="text-naama-blue-200 font-sans-nunito mt-3 text-center">
```

tingtingtingtingtingtingtingtingtingtin commented on src/components/joinus/joincards.tsx:
```diff
@@ -0,0 +1,26 @@
+        <p className="text-naama-blue-200 font-sans-nunito mt-3 text-center font-[400]">
+          {description}
+        </p>
+        <div className="bg-naama-blue-100 h-12 w-12 self-end rounded-full">
```
center the icon within the circle

tingtingtingtingtingtingtingtingtingtin commented on src/app/joinus/page.tsx:
```diff
@@ -0,0 +1,31 @@
+import JoinUsCard from "@/components/joinus/joincards";
+import { HiOutlineUserGroup } from "react-icons/hi2";
+import { Stethoscope } from "lucide-react";
+import { HandHeart } from "lucide-react";
```
```suggestion
import { Stethoscope, HandHeart } from "lucide-react";
```

tingtingtingtingtingtingtingtingtingtin commented on src/app/joinus/page.tsx:
```diff
@@ -0,0 +1,31 @@
+        title="Mentorship and Networking"
+        description="Gain access to a strong network of healthcare professionals and
+        mentors who can guide you on your academic and career path."
+        icon={<HiOutlineUserGroup className="text-naama-blue-300 h-10 w-10" />}
```
```suggestion
        icon={<HiOutlineUserGroup" />}
```

move these to inside the component as all of them are the same

tingtingtingtingtingtingtingtingtingtin replied:
same for other JoinUsCards

mmart570 commented:
<img width="2056" alt="Screenshot 2025-04-27 at 1 28 52 PM" src="https://github.com/user-attachments/assets/7deb6da9-59da-448a-b841-a79de28a6d23" />

- removed the separate join us page file and imported into the main src/page.tsx 
  - to do this, i created 2 separate component files: 1 to represent a single card, and 1 to represent the whole card section

tingtingtingtingtingtingtingtingtingtin reviewed (approved):
thank you for the changes!
